src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/GetIndexCommands.sql

```diff
@@ -0,0 +1,120 @@
+﻿--IF object_id('GetIndexCommands') IS NOT NULL DROP PROCEDURE dbo.GetIndexCommands
+CREATE   PROCEDURE [dbo].[GetIndexCommands]
+@Tbl VARCHAR (100), @Ind VARCHAR (200), @AddPartClause BIT, @IncludeClustered BIT, @Txt VARCHAR (MAX)=NULL OUTPUT
```

Comment:
This does not match original stored procedure. Please restore.

Reply:
I have restored it in my new commit.

Reply:
Sorry, I still do not see correct script.